### ISSUE TITLE
Fabric Loader Error 1.18

### ISSUE THREAD

**Issue by jaded33** [itzg/docker-minecraft-server#1142]
### Describe the problem

I had this container working earlier with 1.18 but fabric mods weren't loading.  Started a clean directory and tried manually loading the versions described HERE: https://fabricmc.net/2021/11/30/118.html

I couldn't get the container to start loading the local copies of Fabric Installer 0.9.0 with Fabric Loader 0.12.8.

If I start the container without those manual versions and just as a Fabric server as I ran earlier I now get this error:

[init] Downloading https://maven.fabricmc.net/net/fabricmc/fabric-installer/0.10.1/fabric-installer-0.10.1.jar ...
[mc-image-helper] 00:02:56.262 ERROR : Operation failed: Connection reset
[init] Failed to download from given location https://maven.fabricmc.net/net/fabricmc/fabric-installer/0.10.1/fabric-installer-0.10.1.jar

Using Unraid Docker environment.

### Container definition

Unraid template for the container is attached in txt file.

### Container logs

[init] Resolved version given 1.18 into 1.18
[init] Resolving type given FABRIC
[init] Checking Fabric version information.
[init] Checking Fabric Loader version information.
[init] Downloading https://maven.fabricmc.net/net/fabricmc/fabric-installer/0.10.1/fabric-installer-0.10.1.jar ...
[mc-image-helper] 00:02:56.262 ERROR : Operation failed: Connection reset
[init] Failed to download from given location https://maven.fabricmc.net/net/fabricmc/fabric-installer/0.10.1/fabric-installer-0.10.1.jar

[my-Minecraft-1.18.txt](https://github.com/itzg/docker-minecraft-server/files/7631083/my-Minecraft-1.18.txt)

**Comment by itzg** [comment]
I'm not able to recreate this, but in any case a "connection reset" generally indicates a networking issue somewhere closer to your end. Sometimes those networking issues are temporary, so perhaps try again.

My testing logs FYI:
```
> docker run -it --rm -e EULA=TRUE -e TYPE=FABRIC itzg/minecraft-server
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 2 1000 1000 4096 Nov 17 00:56 /data'
[init] Resolved version given LATEST into 1.18
[init] Resolving type given FABRIC
[init] Checking Fabric version information.
[init] Checking Fabric Loader version information.
[init] Downloading https://maven.fabricmc.net/net/fabricmc/fabric-installer/0.10.1/fabric-installer-0.10.1.jar ...
[init] Installing Fabric 1.18 using fabric-installer-0.10.1.jar with loader version 0.12.8
Loading Fabric Installer: 0.10.1
Installing Fabric Loader 0.12.8(1.18) on the server
```

**Comment by jaded33** [comment]
I can start up vanilla server fine. I can pull the container from the same computer. I can connect to the container locally. I even downloaded the fabric installer and loader jar files put them in the folder and put the appropriate variables in the config but when I run the container it says it can’t find the installer jar I put in there. Checked permissions. 
I never had this problem with any of your dockers I have run on this machine until now. I am spinning up a VM on another machine host and will try there.

**Comment by jaded33** [comment]
I also just pinged 1.1.1.1 from inside the container and connects to internet fine.

**Comment by jaded33** [comment]
I also tried this on an Ubuntu 20.04 fresh VM on a completely different machine and same error.  I have network wide DNS and pihole adblocker but it isn't being blocked there.  Can ping and navigate to fabricmc.net just fine from any machine in network. I can manually download the fabric installer and put it in the docker container and tell it to use that jar but it reports that the file doesn't exist.  I give up as I have tried anything to figure this error out or workaround it.  As the developer cannot reproduce the problem I am stuck.  I have since gotten another server up and running without using this docker.  I will close as there looks to be no resolution to be had from here sadly.
